api/SWR/CarePackage/useCarePackageApi.js

```diff
@@ -21,6 +21,20 @@ const useCarePackageApi = {
       {}
     ),
 
+  careChargeList: ({ status, pageNumber, pageSize = 10, orderByDate, orderBy, modifiedBy }) =>
```

Comment:
should care charges be in this file (this file is 'useCarePackageApi)? @vadim-kirpun

Reply:
No, everything, but `/care-packages` URLs should be moved to corresponding file